client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -411,18 +417,19 @@ class SiteImporterInputPane extends React.Component {
 						) }
 					</div>
 				) }
-				{ this.state.importStage === 'importable' && (
-					<div className="site-importer__site-importer-confirm-site-pane">
-						<SiteImporterSitePreview
-							siteURL={ this.state.importSiteURL }
-							importData={ this.state.importData }
-							isLoading={ this.state.loading }
-							resetImport={ this.resetImport }
-							startImport={ this.importSite }
-							site={ this.props.site }
-						/>
-					</div>
-				) }
+				{ this.state.importStage === 'importable' &&
+					! ( importStarted && ! this.state.loading ) && (
```

Comment:
The negation's a bit hard to follow - would it work to say `( ! importStarted && this.state.loading )`?